ansible/roles/common/tasks/os/redhat-8.yml

```diff
@@ -41,6 +41,11 @@
   command: 'dnf config-manager --set-enabled *appstream* *baseos* *codeready-builder*'
   when: ansible_distribution == "RedHat"
 
+- name: Install epel-gpgkey
+  ansible.builtin.rpm_key:
+    state: present
+    key: "{{ epel_release_8_key }}"
```

Comment:
This variable is not defined for Rocky Linux. I would just remove the variable aspect here entirely and point to the URI.